src/stylesheets/conversation.less

```diff
@@ -190,5 +182,4 @@
             }
         }
     }
-}
-
+}
```

Comment:
@lemieux Doesn't the new csscomb add those usually? Are we all using the same formatter?